js/controller.js

```diff
@@ -1214,9 +1217,10 @@ OO.plugin('Html5Skin', function(OO, _, $, W) {
       this.state.seeking = false;
       if (this.state.queuedPlayheadUpdate) {
         OO.log('popping queued update');
-        this.skin.updatePlayhead.apply(this.skin, this.state.queuedPlayheadUpdate);
-        this.state.queuedPlayheadUpdate = null;
-        this.renderSkin();
+        this.skin.updatePlayhead.apply(this.skin, this.state.queuedPlayheadUpdate).then(() => {
```

Comment:
The context of the this.skin.updatePlayhead could be binded shorter with arrow function or with bind in the class constructor. I vote for arrow function.

Reply:
\+

Reply:
I did not get your point. 
As I understood method apply is used here because of next argument is array.